.github/workflows/deploy.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: withastro/action@v2
         with:
           path: ./docs
-          package-manager: yarn@latest
+          package-manager: pnpm@latest
```

Comment:
🟡 @crherman7 could you please help me with this? I'm not sure I can test it

Reply:
https://github.com/withastro/action?tab=readme-ov-file#inputs this looks right and I guess technically we don't even really need `latest` according to docs - but doesn't hurt have just so people do know it is the latest.

I will likely just need to run this later and fix as necessary :)